File: tests/perf/elixir/regex_capture.exs
rx = ~r/([A-Za-z]+|_+)[0-9]{2,3}/
s = "xxab123--__99--zz7"
acc = Enum.reduce(1..200_000, 0, fn _, acc ->
  [m | _] = Regex.run(rx, s)
  acc + byte_size(m)
end)
IO.puts(acc)

File: tests/perf/python/regex_capture.py
import re

rx = re.compile(r"([A-Za-z]+|_+)[0-9]{2,3}")
s = "xxab123--__99--zz7"
acc = 0
for _ in range(200_000):
    m = rx.search(s)
    acc += len(m.group(0))
print(acc)
